src/config.js

```diff
@@ -203,6 +203,41 @@ export const BENCHMARKS = {
     labels: ['Firefox (tiering)', 'Firefox (wasm-baseline)', 'Firefox (wasm-ion)', 'Chrome'],
     label: 'WebAssembly Embenchen',
   },
+  'wasm-godot': {
+    compare: {
+      'raptor-wasm-godot-firefox': {
+        color: '#e55525',
+        label: 'Firefox (tiering)',
+        frameworkId: RAPTOR_FRAMEWORK_ID,
+        suite: 'raptor-wasm-godot-firefox',
+        buildType: 'opt',
+      },
+      'raptor-wasm-godot-baseline-firefox': {
+        color: 'red',
+        label: 'Firefox (wasm-baseline)',
+        frameworkId: RAPTOR_FRAMEWORK_ID,
+        suite: 'raptor-wasm-godot-baseline-firefox',
+        buildType: 'opt',
+      },
+      'raptor-wasm-godot-ion-firefox': {
+        color: 'brown',
+        label: 'Firefox (wasm-ion)',
+        frameworkId: RAPTOR_FRAMEWORK_ID,
+        suite: 'raptor-wasm-godot-ion-firefox',
+        buildType: 'opt',
+      },
+      'raptor-wasm-godot-chrome': {
```

Comment:
We won't be able to have Chrome data until issue #97 is tackled.
We can leave the config in here and once #97 is fixed we will see the results.

Reply:
Sounds fine to me!